bin/run-standard-tests.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -eo pipefail
+
+display_and_run() {
+    echo "***" "$@"
+	eval $(printf '%q ' "$@")
+}
+
+# reset workdir to state from git (to remove possible rewritten dependencies)
+display_and_run git reset --hard
```

Comment:
:hairy_eyebrow:

This seems to be for 'gx' detritus that doesn't apply to our world here...